lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
+     *
+     * The $rootElement must be specified in clark notation.
+     *
+     * @param string $rootElementName
+     * @param object $popo
+     * @param string $namespace
+     */
+    function mapPopo($rootElementName, $popo, $namespace) {
+        if (!is_object($popo)) {
```

Comment:
I am fine with it. Do you have a use case for the classnames already or is it just hypothetical?

In case there is not use case yet I would prefer waiting for one